synthetic_data/distinct_generators/text_generator.py

```diff
@@ -10,7 +10,7 @@ def random_string(
     rng: Generator,
     chars: Optional[List[str]] = None,
     str_len_min: int = 1,
-    str_len_max: int = 256,
+    str_len_max: int = 1000,
```

Comment:
I think we should clarify the docstring so its clear the value here is one above the largest per the [docs](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.integers.html#:~:text=If%20provided%2C%20one%20above%20the%20largest%20(signed)%20integer%20to%20be%20drawn%20from%20the%20distribution%20(see%20above%20for%20behavior%20if%20high%3DNone).%20If%20array%2Dlike%2C%20must%20contain%20integer%20values)